frontend/test/parsing/testParseAttributes.cpp

```diff
@@ -334,14 +431,16 @@ static void test4(Parser* parser) {
   assert(modAttr);
   assert(modAttr->isDeprecated());
   assert(modAttr->deprecationMessage() == "Module is deprecated");
-  assert(modAttr->hasPragma(PRAGMA_NO_DOC));
+  // assert(modAttr->hasPragma(PRAGMA_NO_DOC));
+  auto noDoc = modAttr->getAttributeNamed(UniqueString::get(parser->context(), "chpldoc.nodoc"));
```

Comment:
This makes me wish we just had implicit conversion from `const char*` to `UniqueString`.